lib/bundler/mirror.rb

```diff
@@ -117,7 +118,7 @@ class MirrorConfig
 
       def initialize(config_line, value)
         uri, fallback =
-          config_line.match(%r{^mirror\.(all|.+?)(\.fallback_timeout)?\/?$}).captures
+          config_line.match(%r{\Amirror\.(all|.+?)(\.fallback_timeout)?\/?\z}).captures
```

Comment:
preference -- I don't like using `^$` since they're relative to a line, rather than the whole string